packages/shared/lib/services/providers.ts

```diff
@@ -23,6 +24,48 @@ export function getProvider(providerName: string): Provider | null {
     return providers?.[providerName] ?? null;
 }
 
+export async function monitorProviders(): Promise<void> {
+    const providersUrl = process.env['PROVIDERS_URL'];
+
+    // fall back to standard disk loading if no URL is provided
+    if (!providersUrl) {
+        return;
+    }
+
+    const maybeReloadInterval = process.env['PROVIDERS_RELOAD_INTERVAL'];
+    const reloadInterval = parseInt(maybeReloadInterval ?? '') || 60000;
```

Comment:
if you use `parseEnvs(ENVS)` from the `utils` package you'll get all the this for free

Reply:
Fixed in e70ed3f7